libraries/AP_Arming/AP_Arming.h

```diff
@@ -33,22 +33,22 @@ class AP_Arming {
         ARMING_CHECK_MISSION    = 0x4000,
     };
 
-    enum ArmingMethod {
+    enum class Method {
```

Comment:
Sorry to be annoying, and I'm just one voice of course but I actually prefer "ArmingMethod" to just "Method".  I think "Method" is almost a reserved word.  Same goes for "ArmingRequired" vs "Required".  It's a small thing and I'd like to hear another opinion perhaps from @WickedShell.

Reply:
Anything not in the arming namespace has to prefix it with AP_Arming.  Anything within the arming namespace *can* include the namespacing.  The "Arming" in "ArmingMethod" is redundant.

Would you be happy enough if w used the namespacing in the AP_Arming methods too?